demo-app/src/pages/ot/playground/index.tsx

```diff
@@ -9,9 +9,7 @@ workflow:
       view:
         component: OTAvailableWorkflowsFromEndpoint
         props:
-          link:
-            type: component
```

Comment:
What other types can you envision in the future? Some that come to mind are
- enum
- function
- component with args

Part of me is a bit concerned that by removing the type, it makes it a bit more fragile as we have to parse values to know what type it is.

Reply:
Of those three I don't know how to handle functions, but enums are just strings and a component with args should probably be done as children with sub frames.

I think I agree with you conceptually: giving up types feels a bit wrong. But in practice I'm not sure we've given much or anything up. Asking if something has `type === 'string'` is the same work as `typeof prop === 'string'`. Yaml natively supports strings, numbers (ints, floats), dates, and booleans so we can use standard `typeof`.

But again I don't know how to handle functions yet and that will be important. For now I'd be down to try it this way and see if it works with the zod stuff you're going to add in.